### ISSUE TITLE
Unable to run the script

### ISSUE THREAD

**Issue by SuperUserockx** [rluisr/nicehash_exporter#1]
Upon setting the API ID and KEY the script is not sunning and giving a parsing error.

```
 ./run.sh | sponge /home/ubuntu/nicehash_exporter/data/nicehash.prom
parse error: Invalid numeric literal at line 1, column 11
```

![image](https://user-images.githubusercontent.com/18094331/137620536-c1b43a56-1d93-48e2-a203-27033eed335f.png)

**Comment by rluisr** [comment]
Please check `KEY`, `SEC` and `ORG` are correct.

**Comment by SuperUserockx** [comment]
Correct me if i am wrong.

KEY is the API key 
SEC is the secret for the said API key
ORG is the organization ID.

For my case i am not running in an organization but on my personal acc directly, hence i have left it blank.

I am available as SuperUserockx#2729 on discord
